### PR TITLE
Link to Settings

### LINKED ISSUE #32: Link to Settings from ContentUnavailableView

### PR DESCRIPTION
**Pull request by pteasima**
Closes #32

### CODE REVIEW

**Comment by tillkruss** [comment]
This needs a rebase.

File: Ruddarr/Views/MoviesView.swift [hunk 1]
```diff
@@ -24,6 +28,15 @@ struct MoviesView: View {
 
         NavigationStack {
             Group {
+                ContentUnavailableView(
```
Comment (tillkruss) [inline Ruddarr/Views/MoviesView.swift:31]: Why is this up here and not in line 66? Is that because of the commits to `main`?

Comment (pteasima) [reply Ruddarr/Views/MoviesView.swift:31]: cause Im an idiot, is that a good explanation? 😀

Was testing this when I didn't have an instance, never removed it, will do now.

Comment (pteasima) [reply Ruddarr/Views/MoviesView.swift:31]: *when I had an instance and didn't wanna reset everything

**Comment by pteasima** [comment]
rebased, ready to merge imho

**Pull request merged**